know/rdf/parser/sparql_reader.py

```diff
@@ -2,16 +2,16 @@
 import os
 
 
-def read_sparql(query_file: str, uri: str) -> str:
-    """ Reads the .sparql file and replaces $0 with the uri """
+def read_sparql(query_file: str, q_id: str) -> str:
+    """ Reads the .sparql file and replaces $0 with "Q" + entity id """
```

Comment:
Calling it `q_id` basically implies that all ID's will start with `q`, which may not be the case for sites like dbpedia. As such, something like `entity_id` or something would be better. Would probably be worth noting in the docstring that for wikidata, it starts with a Q though.